tools/autograd/derivatives.yaml

```diff
@@ -1234,6 +1234,21 @@
   self: slice_backward_wrapper(grad, self.sizes(), dim, start, end, step)
   result: auto_linear
 
+- name: slice_scatter(Tensor self, Tensor mutated_view, int dim=0, int? start=None, int? end=None, int step=1) -> Tensor
+  self: grad * slice_scatter(at::ones(self.sizes(), self.options()), at::zeros(mutated_view.sizes(), mutated_view.options()), dim, start, end, step)
+  mutated_view: grad * slice_scatter(at::zeros(self.sizes(), self.options()), at::ones(mutated_view.sizes(), mutated_view.options()), dim, start, end, step)
```

Comment:
That is not the right shape?
This should just be a slice no?

Reply:
How come gradcheck didn't catch this?

Reply:
Because it looks like a broadcasted argument I think :'(
And so the engine will reduce the gradient to match the shape of `mutated_view`...

Reply:
Just to confirm - I think this should just be a tensor of ones, of the size of `mutated_view`.